 M docker-compose.yml
 M frontend/package.json
 M mongodb_scripts/backup.sh
?? backups/
?? mongodb_data/
diff --git a/docker-compose.yml b/docker-compose.yml
index 400dff8..c2d4cc5 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -6,7 +6,7 @@ services:
       context: ./frontend
       dockerfile: Dockerfile
       args:
-        - REACT_APP_API_URL=https://music-put.lab.dariah.pl/ethnopedia-api/
+        - REACT_APP_API_URL=https://ethnopedia-api.ckc.uw.edu.pl/api/
     ports:
       - "4050:80"
     volumes:
@@ -54,4 +54,5 @@ services:
     restart: always
 
 volumes:
-  mongodb_data:
\ No newline at end of file
+  mongodb_data:
+
diff --git a/frontend/package.json b/frontend/package.json
index f332e38..de97261 100644
--- a/frontend/package.json
+++ b/frontend/package.json
@@ -33,7 +33,7 @@
         "yup": "^1.3.2"
     },
     "main": "main.js",
-    "homepage": "/ethnopedia",
+    "homepage": "/",
     "scripts": {
         "start": "react-scripts start",
         "build": "react-scripts build",
diff --git a/mongodb_scripts/backup.sh b/mongodb_scripts/backup.sh
index c7205b7..595aa67 100755
--- a/mongodb_scripts/backup.sh
+++ b/mongodb_scripts/backup.sh
@@ -2,8 +2,8 @@
 set -e
 
 # Variables
-SERVER_PREFIX="music_put_lab_dariah_pl"
-#SERVER_PREFIX="ckc_uw_edu_pl"
+#SERVER_PREFIX="music_put_lab_dariah_pl"
+SERVER_PREFIX="ckc_uw_edu_pl"
 #SERVER_PREFIX="metadata_music_put_lab_dariah_pl"
 DUMP_DIR="mongodb_data/dump"
 TIMESTAMP=$(date +"%Y-%m-%d_%H-%M-%S")
